packages/new-polymath-ui/src/components/inputs/TextInput/TextInput.tsx

```diff
@@ -34,7 +37,18 @@ TextInputPrimitive.defaultProps = {
   value: '',
 };
 
-const EnhancedTextInput = formikProxy(TextInputPrimitive);
-export const TextInput = Object.assign(EnhancedTextInput, {
+const EnhancedDatePickerInput: FC<FormikExternalProps> = ({
```

Comment:
Shouldn't this be called `EnhancedTextInput`?

Reply:
Whoops